hedera-node/src/main/java/com/hedera/services/store/models/OwnershipTracker.java

```diff
@@ -73,10 +73,18 @@ public static Change fromMinting(Id treasury, long serialNumber) {
 		return change;
 	}
 
+	public static Change fromWiping(Id accountId, long serialNumber) {
+		var change = new Change();
+		change.setPreviousOwner(accountId);
+		change.setNewOwner(Id.DEFAULT);
+		change.setSerialNumber(serialNumber);
+		return change;
+	}
+
 	public static Change fromBurning(Id treasury, long serialNumber) {
 		Change change = new Change();
-		change.setNewOwner(Id.DEFAULT);
 		change.setPreviousOwner(treasury);
+		change.setNewOwner(Id.DEFAULT);
 		change.setSerialNumber(serialNumber);
 		return change;
 	}
```

Comment:
I think that we can use only one method for assembling the `Change`. Maybe it would be better to rename `fromMinting` and `from..` to:
- `forMinting` 
- `forRemoval`

So when you do `ownershiptracker.forMinting` or `ownershiptracker.forRemoval`